.travis.yml

```diff
@@ -43,9 +43,10 @@ script:
     echo "Running ACNV pipeline";
     bash scripts/acnv_cromwell_tests/run_acnv_pipelines.sh;
   elif [[ $TEST_DOCKER == true ]]; then
-    echo "Building docker image with unit tests...";
-    cd scripts/docker/;
-    sudo bash build_docker.sh  -e `git rev-parse ${TRAVIS_BRANCH}` -s;
+    echo "Building docker image with unit tests..." ;
+    echo "${TRAVIS_BRANCH}" ;
+    sudo cp -rfp scripts/docker/Dockerfile . ;
+    sudo docker build -t broadinstitute/gatk-protected:${TRAVIS_BRANCH} --build-arg GITHUB_TAG=${TRAVIS_BRANCH} --build-arg GITHUB_DIR=\  . ;
```

Comment:
Ok I just looked -- then build script is unnecessary because it's all stuff you can do from docker commandline. But given the old Dockerfile that this PR replaces it makes sense because you don't move the Dockerfile to root and thus you git clone the repo before building. If you move the new Dockerfile to root, as suggested, and then run regular docker commands it works. That's the preferred way to work on it, but it's fine to leave the Dockerfile where it is now just with the caveat that you have to move it to root before doing a build. Jenkins does all this in the background automatically. 

So it's your call: if you want to keep using the build script, I have to alter it a bit to keep up with the workflow that is now assumed and works in Jenkins.

This is why I had two different Dockerfiles originally in this PR -- I wanted to support your legacy functions.